.gitignore

```diff
@@ -38,11 +38,15 @@ gunicorn.pid
 coverage_report/
 .coverage
 
-#virtualenv
+# virtualenv
 venv/
 .vscode/
 .idea/
 .mypy_cache/
 monkeytype.sqlite3
 .pytype/
 .env
+
+# pipenv
+Pipfile
+Pipfile.lock
```

Comment:
Let's update the README.md instructions to be not with Pipenv, but rather Pip. This is because we have dropped support for Pipenv.

cc/ @canihavesomecoffee

Reply:
Okay, instead of .gitignore, I'll update the README.md.

Reply:
Please update the README and request for review 🙇🏽

Reply:
But for testing, it would be better if we do it in a virtual environment to stay away from python version and other conflicts. Also tests still work with pipenv environment, so should I remove Pipenv and provide other alternatives for creating venv?